### ISSUE TITLE
Page Validate/Unvalidate

### ISSUE THREAD

**Issue by trustmaster** [Cotonti/Cotonti#119]
This was originally core hack from Kilandor. I think this should be included in core. The whole topic on this problem is here:

http://neocrome.net/forums.php?m=posts&q=24047

and the kilandor's solution is this:

http://neocrome.net/forums.php?m=posts&p=124646#124646

What is it meant to do?
Copy paste from forums :

"For example, you click on your validation queue and you have like 10 pages waiting for validation.

You can validate it straight from there - it has "Validate" button for every entry, and "Edit" button as well.

BUT

Let's say you want to view the page first, so you can decide what to do with it - edit it, delete et. You'll click on edit button.

And when you're done editing, you "update" page (it returns you to the page that's submitted). And up there you have links like "Put in validation queue" and "Edit".

Put in a validation queue a page that is already in validation queue? nonsense
This link is only usefull if you're viewing already published page.

So you need to go back to validation queue, find the page that is submitted and then click on Validate!

It's easily done when you have like 2-3 pages which all need to be validated.. But having 15+ pages (some of them need to be published today, some of them in few days and so one)..

Basically, to conclude this long post:

Koradhil's hack recognizes IF page is already validated. If it's validate you will have "Put In Validation Queue"

IF it's not validate, instead of "Put In Validation Queue", there will be "Validate" link.. "
